test/unit-tests/textMatch.test.js

```diff
@@ -1,11 +1,20 @@
 let { openBrowser, goto, closeBrowser, text, textBox, toRightOf, evaluate } = require('../../lib/taiko');
 let { createHtml, removeFile, openBrowserArgs } = require('./test-util');
+let test_name = 'textMatch';
 
-describe('text match', () => {
-
+describe('match', () => {
```

Comment:
updating test to do goto only once reduces a lot of test runtime since goto adds 1s for every testcase .. will be great to have it the same way